website/docs/docs/building-a-dbt-project/using-sources.md

```diff
@@ -164,10 +164,10 @@ Additionally, the `loaded_at_field` is required to calculate freshness for a tab
 These configs are applied hierarchically, so `freshness` and `loaded_at` field values specified for a `source` will flow through to all of the `tables` defined in that source. This is useful when all of the tables in a source have the same `loaded_at_field`, as the config can just be specified once in the top-level source definition.
 
 ### Snapshotting source freshness
```

Comment:
Do we want to change the wording around 'snapshotting' here now that we've removed snapshot from the command name?

Reply:
yes, well spotted - changed the title in https://github.com/dbt-labs/docs.getdbt.com/pull/831/commits/fb40346c5952eb505efeca154d5cd6cb2e108a9d